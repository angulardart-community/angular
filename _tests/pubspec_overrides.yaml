# melos_managed_dependency_overrides: ngast,ngcompiler,ngdart,ngforms,ngtest
dependency_overrides:
  ngast:
    path: ../ngast
  ngcompiler:
    path: ../ngcompiler
  ngdart:
    path: ../ngdart
  ngforms:
    path: ../ngforms
  ngtest:
    path: ../ngtest
