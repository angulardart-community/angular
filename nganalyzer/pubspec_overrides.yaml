# melos_managed_dependency_overrides: ngast
dependency_overrides:
  ngast:
    path: ../ngast
