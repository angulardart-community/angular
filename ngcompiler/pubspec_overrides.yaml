# melos_managed_dependency_overrides: ngast,ngdart
dependency_overrides:
  ngast:
    path: ../ngast
  ngdart:
    path: ../ngdart
