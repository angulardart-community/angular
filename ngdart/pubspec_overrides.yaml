# melos_managed_dependency_overrides: ngast,ngcompiler
dependency_overrides:
  ngast:
    path: ../ngast
  ngcompiler:
    path: ../ngcompiler
