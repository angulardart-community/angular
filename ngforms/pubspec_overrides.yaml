# melos_managed_dependency_overrides: ngast,ngcompiler,ngdart,ngtest
dependency_overrides:
  ngast:
    path: ../ngast
  ngcompiler:
    path: ../ngcompiler
  ngdart:
    path: ../ngdart
  ngtest:
    path: ../ngtest
