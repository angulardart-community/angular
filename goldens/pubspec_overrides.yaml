# melos_managed_dependency_overrides: ngast,ngcompiler,ngdart
dependency_overrides:
  ngast:
    path: ../ngast
  ngcompiler:
    path: ../ngcompiler
  ngdart:
    path: ../ngdart
